onnxruntime/core/framework/session_options.cc

```diff
@@ -32,8 +34,8 @@ Status SessionOptions::AddConfigEntry(_In_z_ const char* config_key, _In_z_ cons
     return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "Config key is empty or longer than maximum length 128");
 
   std::string val(config_value);
-  if (val.length() > 1024)
-    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "Config value is longer than maximum length 1024");
+  if (val.length() > SESS_CONFIG_VALUE_MAX_LEN)
+    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "Config value is longer than maximum length 10240");
```

Comment:
It's good to use to the same const in the error msg.

Reply:
Done.